identity/src/main/java/com/stripe/android/identity/states/IdentityScanState.kt

```diff
@@ -35,7 +35,11 @@ internal sealed class IdentityScanState(val type: ScanType, isFinal: Boolean) :
     /**
      * Initial state when scan starts, no documents have been detected yet.
      */
-    internal class Initial(type: ScanType) : IdentityScanState(type, false) {
+    internal class Initial(
+        type: ScanType,
+        private val transitioner: IdentityFoundStateTransitioner = IOUTransitioner()
```

Comment:
in the future, we should look into using dependency injection for this kind of field

Reply:
yes! will bring dagger in once the SDK is feature completed

Reply:
Create [this](https://jira.corp.stripe.com/browse/IDPROD-3540) internal ticket to track